packages/contractkit/src/kit.ts

```diff
@@ -75,6 +75,7 @@ export interface NetworkConfig {
 export interface KitOptions {
   gasInflationFactor: number
   gasPrice: string
+  gasPriceSuggestionMultiplier: number
```

Comment:
I would add a TODO here too, for removing it when all the nodes support the gasPrice endpoint